lib/fluent/plugin/in_tail.rb

```diff
@@ -113,6 +113,8 @@ def initialize
     config_param :path_timezone, :string, default: nil
     desc 'Follow inodes instead of following file names. Guarantees more stable delivery and allows to use * in path pattern with rotating files'
     config_param :follow_inodes, :bool, default: false
+    desc 'Maximum length of line. The longer line is just skipped.'
+    config_param :maximum_line_limit_size, :size, default: nil
```

Comment:
Including both `maximum` and `limit` seems redundant. How about `max_line_size` or `line_size_limit`?

Reply:
Adopted the former, thanks!